source/renderer/app/components/voting/widgets/PatchCheckbox.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import React from 'react';
+import type { ComponentType, Element } from 'react';
+
+import { Checkbox } from 'react-polymorph/lib/components/Checkbox';
+
+import stylesOverride from './PatchCheckbox.scss';
+
+type Props = {
+  checked: boolean,
+  className?: string,
+  disabled?: boolean,
+  label?: string | Element<any>,
+  labelLeft?: string | Element<any>,
+  labelRight?: string | Element<any>,
+  onChange?: Function,
+  onBlur?: Function,
+  onFocus?: Function,
+  skin?: ComponentType<any>,
+  theme?: ?Object, // will take precedence over theme in context if passed
+  themeId?: string,
+  themeOverrides?: Object,
+};
+
+export function PatchCheckbox(props: Props) {
```

Comment:
@lucas-barros why did we name this component "Patch" checkbox?

Reply:
@nikolaglumac  This is a temporary solution to fix the checkmark position on the voting screen. 

There's a strange behavior in the checkbox component that causes the checkmark to have a slight different offset depending on the screen you are viewing. We decided to patch this for now instead of doing a global fix and avoid regression testing. More details can be found in the tech debt ticket DDW-811.

Do you have another suggestion for the component name?

Reply:
Got it! Thanks for creating DDW-811 - god job!